Our.Umbraco.TagHelpers/ImgTagHelper.cs

```diff
@@ -170,14 +170,28 @@ public override void Process(TagHelperContext context, TagHelperOutput output)
                 }
                 else
                 {
-                    // Pull through an image based on the given width and calculate the height so it's to scale.
-                    imgSrc = MediaItem.GetCropUrl(width: (int)width);
-                    if (_globalSettings.OurImg.LazyLoadPlaceholder.Equals(ImagePlaceholderType.LowQualityImage))
+                    if (ImgHeight > 0)
                     {
-                        // Generate a low quality placeholder image if configured to do so
-                        placeholderImgSrc = MediaItem.GetCropUrl(width: (int)width, quality: _globalSettings.OurImg.LazyLoadPlaceholderLowQualityImageQuality);
+                        imgSrc = MediaItem.GetCropUrl(width: (int)ImgWidth, height: (int)ImgHeight);
+                        if (_globalSettings.OurImg.LazyLoadPlaceholder.Equals(ImagePlaceholderType.LowQualityImage))
```

Comment:
It might be more reabable to set `globalSettings.OurImg.LazyLoadPlaceholder.Equals(ImagePlaceholderType.LowQualityImage)` to a variable. Not a massive deal though!

Reply:
Good catch as this is used a couple of times 👍